pages.id/linux/halt.md

```diff
@@ -1,23 +1,23 @@
 # halt
 
-> Hentikan sistem.
+> Hentikan seluruh proses dan jalannya CPU dalam sistem komputer.
 > Informasi lebih lanjut: <https://manned.org/halt.8>.
 
-- Hentikan sistem:
+- Hentikan sistem komputer:
 
 `halt`
 
 - Matikan sistem (sama seperti `poweroff`):
 
 `halt --poweroff`
 
-- Reboot sistem (sama seperti `reboot`):
+- Nyalakan ulang sistem (sama seperti `reboot`):
 
 `halt --reboot`
 
-- Berhenti segera tanpa menghubungi manajer sistem:
+- Hentikan sistem secara segera tanpa menghubungi manajer sistem:
 
-`halt --force --force`
+`halt --force`
```

Comment:
Just went through the man page, seems like it would work with just one force. Can you update the same on the English page too?

Reply:
Yes, I am planning for this

Reply:
Thanks.